nrovalidate/main.go

```diff
@@ -70,15 +91,11 @@ func parseArgs(args ...string) (ProgArgs, error) {
 		return pArgs, err
 	}
 
-	if pArgs.Version {
-		return pArgs, err
-	}
-
 	return pArgs, nil
 }
 
 func validateCluster(args ProgArgs) error {
-	cli, err := clientutil.New()
+	cli, err := NewClientWithScheme(scheme)
```

Comment:
what are the advantages of using the clientset with the scheme instead of clientset out of clientutil?

Reply:
we need to have extra schemas registered in the client (NRO, MCP) which aren't registered by default in the clientutil one.